.github/workflows/test-suite.yml

```diff
@@ -15,26 +15,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
```

Comment:
I think you need to quote `3.10`, because at the moment this is getting cast to `3.1` and that makes the Github actions fail.